incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -293,10 +295,21 @@ func (r *ConfigReconciler) forceInitialReconcile(log logr.Logger, reason string)
 
 // SetupWithManager builds a controller with the reconciler.
 func (r *ConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// Whenever a CRD is created/updated, we will send a request to reconcile the
+	// singleton again, in case the singleton has configuration for the resource.
+	crdMapFn := handler.ToRequestsFunc(
+		func(a handler.MapObject) []reconcile.Request {
+			return []reconcile.Request{
+				{NamespacedName: types.NamespacedName{
+					Name: api.HNCConfigSingleton,
+				}},
```

Comment:
Nit: this looks a bit unbalanced to me. Maybe just split it into two lines?

```
nnm := types.NamespacedName{Name: api.HNCConfigSingleton}
return []reconcile.Request{{NamespacedName: nnm}}
```

Reply:
Sure!